content/cpp/concepts/methods/methods.md

```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Methods'
-Description: 'C++ class methods are user-defined functions that can be used within an instance of the class. A dot notation is used before method names to distinguish them from regular functions.'
+Description: 'C++ class methods are functions defined within a class, using dot notation to access and manipulate class members and behavior.'
```

Comment:
This description is mostly good, it can just be improved a little bit to make it easy to read, as follows -

```suggestion
Description: 'C++ class methods are functions defined within a class and can be accessed using dot notation to manipulate class members and behaviour.'
```